tests/framework/extensions/users/users.go

```diff
@@ -220,7 +220,13 @@ func AddClusterRoleToUser(rancherClient *rancher.Client, cluster *management.Clu
 		FieldSelector:  "metadata.name=" + cluster.ID,
 		TimeoutSeconds: &defaults.WatchTimeoutSeconds,
 	}
-	watchInterface, err := rancherClient.GetManagementWatchInterface(management.ClusterType, opts)
+
+	adminClient, err := rancher.NewClient(rancherClient.RancherConfig.AdminToken, rancherClient.Session)
```

Comment:
Why isn't this same change in the v2.7 branch?

Reply:
Ah this from the PSP PR right? So you're adding the necessary changes.

Reply:
yes that is correct. Only adding the required changes here and not adding the changes required for PSA RBAC in here.

Reply:
Thank you!